File: AttendanceCalc.py
days = int(input("Total No. of Days: "))
present_days = int(input("No. of Present: "))

percentage_wanted = float(input("Percentage wanted: "))

current_percentage = (present_days / days) * 100
days_required = 0

if percentage_wanted > 100:
    print("Percentage cannot be greater than 100.")
elif current_percentage >= percentage_wanted:
    print("You already meet or exceed the desired percentage.")
else:
    while (present_days / days) * 100 < percentage_wanted:
        present_days += 1
        days += 1
        days_required += 1

    print(f"{days_required} more days")

File: AttendanceCalc.cpp
#include<iostream>

using namespace std;

int main(){
    int days, present, days_required=0;
    float percentage_wanted, current_percentage;
    cout<<"Enter No. of Days: ";
    cin>>days;
    cout<<"Enter No. of Present: ";
    cin>>present;

    current_percentage=present/days;

    cout<<"Percentage wanted: ";
    cin>>percentage_wanted;

    if (percentage_wanted>=100){
        cout<<"INVALID!";
    }
    if (percentage_wanted<=current_percentage){
        cout<<"Already achieved";
    }
    else {
        while((present/(float)days)*100 < percentage_wanted){
            present++;
            days++;
            days_required++;
        }
    }

    cout<<"Number of classes required: "<<days_required;
}
